src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <div class="logo">
+        <a href="/#home">
+          <img
+            src="./images/logo.png"
+            alt="Company logo"
+        />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="list">
+          <li class="list_link">
+            <a
+              href="apple.html"
+              class="nav_link nav_link1 is_active"
+            >
+              apple
+            </a>
```

Comment:
The `href` attribute for internal navigation should start with `#`. The current `href` value 'apple.html' suggests a navigation to another page, which might not be the intended behavior.